help/include/send-group-dm.md

```diff
@@ -2,7 +2,8 @@
 
 {tab|desktop-web}
 
-1. Click the **New direct message** button at the bottom of the app, or
+1. Click the **New direct message** button at the bottom of the app or the
+   **Start new conversation** button if you are in a direct message view, or
```

Comment:
The two "or"s feel like a little much. Should we move the keyboard shortcut into a keyboard tip?